sqliterepo/internals.h

```diff
@@ -89,6 +89,9 @@ struct sqlx_repository_s
 	sqlx_repo_change_hook change_callback;
 	gpointer change_callback_data;
 
+	sqlx_repo_send_event_hook send_event_callback;
+	gpointer send_event_callback_data;
```

Comment:
sqliterepo does not known the event system exist. It cannot decide if an event must be sent or not. The only thing it can do is inform the upper-level that something has changed (in the admin table).